### ISSUE TITLE
[Feature Request] Use Bambu Handy in Lan Mode

### ISSUE THREAD

**Issue by BambuLabDiscord** [bambulab/Bambu-Handy#138]
Being able to start prints from the SD card or monitor and stop prints using Bambu Handy when the printer is in LAN mode would be very useful. I typically keep my printer in LAN mode As I don't have a need to leave it connected to the cloud all the time.

**Comment by Starbase12** [comment]
That would be really great if we can use the mobile phone app without a cloud connection!

**Comment by githubber4ever** [comment]
This feature world really be appreciated since the Bambu Studio can also access the printer in LAN mode.

**Comment by vordenken** [comment]
Yes please add this feature. I like the idea of printing something on the go (without the need of slicing on my computer) but don't want to use all this cloud stuff. 
I am connected to my home network via VPN already so no need for cloud features.

For now I'm using lan mode and the homeassistant integration to check the webcam and stuff.

**Comment by jwjames83** [comment]
Adding another "vote" for getting this implemented sooner rather than later.

**Comment by aspdigital** [comment]
Another vote for eliminating the cloud from the connection between Bambu Handy to the printer.

**Comment by denjohn** [comment]
Seems that the cloud is the connection to the cell. So what alternative is there when one wants to  view prints in progress ? The west doesn't trust this app thinking it's Chinese.  It's not.  Bambu is Japanese.  Please keep this cloud.

**Comment by vordenken** [comment]
> Seems that the cloud is the connection to the cell. So what alternative is there when one wants to  view prints in progress ? The west doesn't trust this app thinking it's Chinese.  It's not.  Bambu is Japanese.  Please keep this cloud. 

It's not always black and white. You can keep the cloud option but just add lan only as well. Their printers/bambuStudio already can do this. 
It's also not only trust, but I want to be sure that I can use the printer like always even if BambuLab goes bankrupt.

**Comment by aspdigital** [comment]
> Seems that the cloud is the connection to the cell. So what alternative is there when one wants to view prints in progress ? The west doesn't trust this app thinking it's Chinese. It's not. Bambu is Japanese. Please keep this cloud.

If my printer and my smartphone are on the same network, there is no need for the machine to send info up to the cloud so the cloud can send it back down to my phone. This has nothing to do with who wrote the app or where their cloud is hosted. It is simply about how there is no need for the cloud step when everything is on the same network.

**Comment by gschintgen** [comment]
This is sorely needed indeed. 

(I have access to a X1C, but this issue is what makes me hesitate to buy a Bambu for myself.)

See also: #80.

**Comment by viktortrass** [comment]
come on guys include this function because we need it!!

**Comment by Jordan87** [comment]
Yes, this request is still valid :)
